faq.rst

```diff
@@ -172,14 +167,11 @@ on how to create a compatible Samba export: `Time Machine backups with Rockstor
 Do you have examples on how to build complete NAS solutions for different storage capacities?
 ---------------------------------------------------------------------------------------------
 
-Rockstor is hardware agnostic, so you can build a complete Linux, BTRFS powered
-NAS solution using the Rockstor NAS OS and hardware of your choice. If you are a
-home-user/prosumer, read `8TB DIY NAS using Rockstor
-<http://rockstor.com/blog/uncategorized/8tb-rockstor-diy-nas/>`_. For a bigger
-storage footprint, read `240TB DIY NAS using Rockstor
-<http://rockstor.com/blog/diy-nas/rockstor-on-45-drives-aka-the-rockinator/>`_
-. The only requirement is that the system be of a 64bit Intel or compatible
-architecture.
+Rockstor is hardware agnostic, so you can build a complete Linux, BTRFS-powered
+NAS solution using the Rockstor NAS OS and hardware of your choice. The only
+requirement is that the system be of a 64bit Intel or compatible architecture.
```

Comment:
We can now add Arm64 to our supported hardware but that change belongs in a seperate pr anyway. Just note here so we have a reference to at least one place we are now out of date re arcitecture.

Reply:
Thanks for the feedback... I had the exact same thought: we can add it, but might be better suited for a separate PR. I'll create an issue appropriately when I get the chance if you do not get to it before me.